lessons/Data-Science/box-plots/langs/en-us/pages/create-bp-from-dp.adoc

```diff
@@ -0,0 +1,62 @@
+= Create Box Plots from Dot Plots
+
+Use the five-number summary to draw a box plot above the corresponding dot plot. When you're finished, identify which quarter(s) of the data are packed the densest, and which quarter(s) of the data are the most dispersed. The first row has been completed as a sample.
+
+[.FillVerticalSpace, cols="^.^1a,<.^5a,^.>15a,^.^5a,^.^5a", options="header"]
+|===
+
+|
+| Five-Number Summary
+| Dot Plot
+| Densest Packed Quarter(s)
+| Most Dispersed Quarter(s)
+
+
+| @n
+| Min: 1 +
+Q1: 3 +
+Median: 6.5 +
+Q3: 9 +
+Max: 20
+|  @image{../images/dp-bp1-soln.png}
+| Q1
```

Comment:
We need to change all of the solutions in these columns to say first, second, third, fourth.
Q1 refers to a quartile, which is the value that divides the first and second quarter of the data.

Reply:
oopsies, good catch. Thanks!